site/app/templates/grading/simple/Display.twig

```diff
@@ -122,7 +122,7 @@
     {% endif %}
 
     {# This is a data table #}
-    <table id="data-table" data-current-grader="{{ user_id }}" class="table table-striped table-bordered persist-area" style="table-layout: fixed;">
+    <table id="data-table" data-gradeable="{{ gradeable.getId() }}" data-current-grader="{{ user_id }}" class="table table-striped table-bordered persist-area" style="table-layout: fixed;">
```

Comment:
Why add data-gradeable?

Reply:
Forgot to delete this change from previous commits, should be good now.